net/strongswan/files/gencerts.sh

```diff
@@ -34,8 +34,8 @@ SHORT_DOMAIN="${DOMAIN%%.*}"
 ORG="$1"; shift
 
 # invariants...
-STRONGSWANDIR=/etc
-SWANCTL_DIR=$STRONGSWANDIR/swanctl
+SYSCONFDIR=/etc
```

Comment:
Forgot to make it default

Reply:
As I said, I'm not convinced that this isn't in the "give them enough rope" category.  But if more people ask me, then I might change it.

What would make more sense is to have variables for CACERTDIR, CERTDIR, KEYDIR instead, since `ipsec` and `swanctl` use different directory names.

Reply:
Alright.